### PR TITLE
Translate into spanish

### PR DESCRIPTION
**Pull request by jesusbibieca**
Translated the README file into spanish. @jzf2101

### CODE REVIEW

**Comment by jzf2101** [comment]
Gracias :) Is this ready for me to review?

**Comment by jesusbibieca** [comment]
Now it should be complying the standard and ready to review @jzf2101

**Comment by Carreau** [comment]
No hablo español, pero apruebo, 

I'll let another speaker press the merge button.  @fperez may be happy to see that and press the green button. @damianavila as well.

File: README.es-ES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Jupyter
+
+*Leer en otros idiomas: [English](README.md), [Español](README.es-ES.md)*
+
+Meta paquetes de Jupyter para su instalacion y documentos.
```
Comment (damianavila) [inline README.es-ES.md]: * `paquete`, sin la `s`
* instalación (con tilde)
* documentación, el lugar de documentos?

File: README.es-ES.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+conda env create -f environment.yml  
+```  
+
+Activa el entorno:
+
+```bash
+source activate jupyter_docs  
+```
+
+Construye los documentos:
```
Comment (damianavila) [inline README.es-ES.md]: Capaz `Genera` en lugar de construye?

File: README.es-ES.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+```bash
+make clean  
+make html
+```
+
+Los documentos se crearan en `build/html`. Estos pueden ser vistos iniciando un servidor HTTP y navegando hasta `0.0.0.0:8000` en tu explorador web.
```
Comment (damianavila) [inline README.es-ES.md]: crearán (con tilde)

Comment (damianavila) [reply README.es-ES.md]: Éstos (con tilde)

**Comment by damianavila** [comment]
I left some comments, thanks for the ping @Carreau.

**Comment by jesusbibieca** [comment]
Thank you for the feedback @damianavila

File: README.es-ES.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# Jupyter
+
+*Leer en otros idiomas: [English](README.md), [Español](README.es-ES.md)*
+
+Meta paquete de Jupyter para su instalación y documentación.
+
+## Ejecutando los documentos localmente
```
Comment (fperez) [inline README.es-ES.md]: Yo aquí tal vez diría "Cómo ejecutar la documentación localmente".

File: README.es-ES.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+```
+
+Genera los documentos:
+
+```bash
+make clean  
+make html
+```
+
+Los documentos se crearán en `build/html`. Éstos pueden ser vistos iniciando un servidor HTTP y navegando hasta `0.0.0.0:8000` en tu explorador web.
```
Comment (fperez) [inline README.es-ES.md]: Mmh.. Si bien esta traducción es 100% correcta, en este caso me parece que deberíamos cambiar el original: para leer documentación de Sphinx no hace falta abrir un nuevo servidor, basta abrir la página `build/html/index.html` con cualquier navegador...

Yo preferiría que corrigiésemos eso en el original, y en la corrección...

Haré un comentario al respecto en ingés.

File: README.es-ES.md (hunk 4 above)
Comment (jesusbibieca) [reply README.es-ES.md]: Que tal "Generar la documentación localmente"?

**Comment by fperez** [comment]
This looks solid, thanks, just one minor fix suggested.

But there's one thing I noticed that I think we should fix in the English version, and therefore would affect this: I don't see the point of suggesting that users start an HTTP server just to read the sphinx docs. As far as I know that's unnecessary, sphinx builds can be read simply by opening `build/html/index.html` with a browser.

Is that not the case here? I've never seen a sphinx  build that requires a full-fledged server to read correctly...

If I'm wrong, then no other objections. If I'm right, let's then simplify the language both in English and Spanish to just "open `build/html/index.html` in your web browser".

**Comment by fperez** [comment]
Thanks @jesusbibieca! I'd like to get confirmation from @jzf2101, @willingc, @Carreau or someone else that this particular sphinx build is really OK to open locally. I'm surprised to see the language about a server there, so it may be there for a good reason and I simply don't know. But if it was accidental, then yes, let's update it.  They have more hands-on experience with that doc build than I do, so I don't want to override anyone, just asking a question before we start propagating that info across translations :)

**Comment by jzf2101** [comment]
@fperez - I believe you're right.  I don't think you actually have to star the server.  I haven't at least.

**Comment by willingc** [comment]
Thanks @jesusbibieca :cake:

@fperez and @jzf2101, There are two ways to view documentation that you have built locally:

1. view the html files in a browser
2. start a webserver as Jesus documented

Either one is correct. If viewing changes on made on one file, I typically use 1. If viewing changes that touch multiple pages or the `conf.py`, I typically start a webserver as in 2.

**Comment by fperez** [comment]
Thx @willingc @jzf2101 for the input, and @jesusbibieca for the contribution! Very happy to see our Spanish content grow :)

**Comment by jesusbibieca** [comment]
You're welcome! @fperez  @willingc  @jzf2101

**Pull request merged**